src/common/options.cc

```diff
@@ -1712,6 +1712,11 @@ std::vector<Option> get_global_options() {
     .add_service("mgr")
     .set_description("Issue a health warning if there are fewer OSDs than osd_pool_default_size"),
 
+    Option("mon_warn_on_slow_ping_time", Option::TYPE_UINT, Option::LEVEL_BASIC)
+    .set_default(1000000)
+    .add_service("mgr")
+    .set_description("Issue a health warning if heartbeat ping longer than specified microseconds"),
```

Comment:
should the option be msec as well since you've changed the output? 
This could be 'advanced' as well, most deployments won't need to set it

Reply:
@jdurgin Do you mean to change the name to something like mon_warn_on_slow_ping_msec?

Reply:
@jdurgin ok, I didn't read your comment right.  You want this in milliseconds instead of microseconds.  The name doesn't matter as much.

Reply:
yeah I meant the value

Reply:
@jdurgin Also, I switched to a ratio of grace, so this defaults to 0 now.  This is out of date code but your comment still applies.